PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -322,9 +322,20 @@ def miniAOD_customizeCommon(process):
 
 
 def miniAOD_customizeMC(process):
+    task = getPatAlgosToolsTask(process)
+    #GenJetFlavourInfos
+    process.load("PhysicsTools.JetMCAlgos.HadronAndPartonSelector_cfi")
+    process.selectedHadronsAndPartons.particles = "prunedGenParticles"
```

Comment:
If selectedHadronsAndPartons is ran with prunedGenParticles as input, it will not get the full tree of particles, but only status=1 and status=23 particles saved on MiniAOD, but not the intermediate particles.
To really use the new physics flavor definition, this would need to take genParticles as input.